Resources/doc/annotations/security.rst

```diff
@@ -15,7 +15,7 @@ The ``@Security`` annotation restricts access on controllers::
     class PostController extends Controller
     {
         /**
-         * @Security("has_role('ROLE_ADMIN')")
+         * @Security("has_role('ROLE_ADMIN')", message="Staff only.")
```

Comment:
I'd probably not show the `message` here, so we can show the simplest usage example (and you do a nice job below of showing how to use it).

Reply:
Removed this one